br/cmd/br/backup.go

```diff
@@ -110,6 +108,8 @@ func NewBackupCommand() *cobra.Command {
 			build.LogInfo(build.BR)
 			utils.LogEnvVariables()
 			task.LogArguments(c)
+			// Do not run stat worker in BR.
+			session.DisableStats4Test()
```

Comment:
Do we need to rename the function name `DisableStats4Test`?

Reply:
rename it in the future  https://github.com/pingcap/tidb/issues/50531